bin/bootstrap.sh

```diff
@@ -14,11 +14,12 @@ has_otomi='false'
 [ -f $ENV_DIR/bin/otomi ] && has_otomi='true'
 
 function generate_loose_schema() {
-  local targetPath="$ENV_DIR/.vscode/values-schema.yaml"
-  local sourcePath="$PWD/values-schema.yaml"
-  yq d $sourcePath '**.required.' | yq d - 'properties.toolsVersion' | yq d - 'properties.cluster' >$targetPath
+  local targetPath="${ENV_DIR}/.vscode/values-schema.yaml"
+  local sourcePath="${PWD}/values-schema.yaml"
```

Comment:
please no unneeded chars

Reply:
Same reason as https://github.com/redkubes/otomi-core/pull/271#discussion_r562634102
Just following the style guide, but I'll remove.